src/vtlengine/Operators/Time.py

```diff
@@ -759,3 +760,22 @@ def evaluate(cls):
         result = cls.validate()
         result.value = date.today().isoformat()
         return result
+
+
+class Month(Operators.Unary):
+    op = 'month'
+
+    @classmethod
+    def py_op(cls, value: str):
+        if '/' in value:
+            raise RuntimeError
+        if '-' in value:
```

Comment:
Need to check if there is a Date here.

Suggestion: use this python code to check if there are two hyphens on same str:

```python
if value.count('-') == 2:
   # Date processing
  .....
```